assignments/array-methods.js

```diff
@@ -56,21 +56,45 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
-console.log(fullName);
+runners.forEach(function(i){
```

Comment:
good job here.  i is often used to point to index, so this could be confusing to some.  Its best to use plain english to say what you are forEaching.  but technically sound.